Source/Core/Common/CDUtils.cpp

```diff
@@ -208,7 +208,7 @@ std::vector<std::string> cdio_get_devices()
 // Returns true if device is a cdrom/dvd drive
 bool cdio_is_cdrom(std::string device)
 {
-#ifndef _WIN32
+#ifdef _WIN32
```

Comment:
Common.h was not included in the original file.  This meant when I saw MAX_PATH, I only saw it as part of Window's libraries versus the extended set provided by Common.h.